iFixit/react-commerce
# Use new collection type names

This PR closes #116 

I'm still keeping the name "collections" in many parts of the codebase mostly to avoid conflicts with other branches that are ready to be reviewed after this PR. When those branches will be merged too then I'll refactor all references to "collection"

## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Verify that product list page works with the new Strapi instance
